lib/manager/actions/status.js

```diff
@@ -194,6 +199,15 @@ export function handleFinishedJob (job: ServerJob) {
         break
       case 'DEPLOY_TO_OTP':
         return dispatch(fetchDeployment(job.deploymentId))
+      // No completion step for build transport network. User should just
+      // re-request isochrones.
+      case 'SYNC_PROJECT_FEEDS':
```

Comment:
How does this action ever get fired?

Reply:
Oi, I think the branch I pulled in from catalogueglobal must have also had some changes related to https://github.com/catalogueglobal/datatools-server/pull/32/files#diff-42c8506cd760bd0551bfefeb52ce5c33R23.